contrib/epee/include/net/abstract_http_client.h

```diff
@@ -0,0 +1,173 @@
+#pragma once
+
+#include <boost/optional/optional.hpp>
+#include "http_base.h"
+#include "http_auth.h"
+#include "net/net_ssl.h"
+#include "net_parse_helpers.h"
+
+#undef MONERO_DEFAULT_LOG_CATEGORY
+#define MONERO_DEFAULT_LOG_CATEGORY "net.http"
```

Comment:
Move to cpp ... I think? @moneromooo-monero

Reply:
I don't know offhand if any show up on the profile.

Reply:
No compile errors when moved to cpp.